src/storyMap/components/StoryMap.js

```diff
@@ -218,16 +284,23 @@ const StoryMap = props => {
 
   const getLayerPaintType = useCallback(
     layer => {
-      const layerType = map.getLayer(layer).type;
+      const layerType = map.getLayer(layer)?.type;
+      if (!layerType) {
+        console.warn(`Layer ${layer} not found`);
```

Comment:
question: should this be removed?

Reply:
I think it should be there, I think I can improve it using the `logger`, this means that a laye added in the configuration doesn't exist in the mapbox style